gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/CodeSystemsUtil.java

```diff
@@ -1,10 +1,13 @@
 package uk.nhs.adaptors.pss.translator.util;
 
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 import org.springframework.stereotype.Component;
 
 import java.util.Map;
 
 @Component
```

Comment:
Can the `@Component` annotation be removed? It doesn't seem to be injected anywhere.

Reply:
Also, the `@Component` annotation identifies the class as a bean. Without a no argument public constructor it technically isn't. So I think we should remove it.

Reply:
Yes, agree, it is not used as a Spring component.